.expeditor/verify_windows_pipeline.yml

```diff
@@ -0,0 +1,10 @@
+  - label: "[lint] :windows: "
+    command:
+      - ./test/not-yet-implemented.ps1
+    expecitor:
+      executor:
+        docker:
```

Comment:
nit: not that it matters at the moment, but in order to run this on windows you'd need to specify `host_os: windows` nested underneath docker.

Reply:
added!